### DISCUSSION TITLE
Bump the go version and Operator version in .devcontainer/Dockerfile

### DISCUSSION THREAD

**Discussion by Shubham82** [kedacore/keda#1610] [General] [answered]
Hi Folks!

Is it legitimate to bump the golang version to 1.15.6 [here](https://github.com/kedacore/keda/blob/main/.devcontainer/Dockerfile#L6) as we used go 1.15.6 in keda. see this [pr](https://github.com/kedacore/keda/pull/1505). and also for the OPERATOR_RELEASE_VERSION  ([releases](https://github.com/operator-framework/operator-sdk/releases/)) in the same file [here](https://github.com/kedacore/keda/blob/main/.devcontainer/Dockerfile#L84) i.e  .devcontainer/dockerfile. 
If it looks good then, i will happy to do it and raise a PR for it.

**Comment by Shubham82** [comment]
Hi @zroubalik  can you please take a look 
thanks.

**Reply by zroubalik** [reply]
Yeah, it should be probably updated. Do you use Devcontainer (I don't), so it needs to be tested before merging into main branch.

**Comment by Shubham82** [comment, marked as answer]
I previously tried it. i have to set-up again, will try this version of go. 
Will update you as soon as i am done with it. but can i also update the OPERATOR_RELEASE_VERSION ? If yes so can you tell which one i will use?
Thanks.

**Reply by zroubalik** [reply]
Should be in sync with https://github.com/kedacore/keda/blob/main/tools/build-tools.Dockerfile#L58

**Reply by Shubham82** [reply]
Ok,  Thanks for the information.

**Reply by Shubham82** [reply]
I checked and DevContainer is created and working for both the updated version of go and operator.
should i raise a pr for it?

**Reply by zroubalik** [reply]
Sure!

**Reply by Shubham82** [reply]
will soon raise a PR for it.
thanks.

**Reply by Shubham82** [reply]
will soon raise a PR for it.
thanks.
